sources/common/infoschema.go

```diff
@@ -253,3 +254,14 @@ func (is *InfoSchemaImpl) GetIncludedSrcTablesFromConv(conv *internal.Conv) (sch
 	}
 	return schemaToTablesMap, nil
 }
+
+// SanitizeDefaultValue removes extra characters added to Default Value in information schema in MySQL.
+func SanitizeDefaultValue(defaultValue string, ty string, generated bool) string {
+	defaultValue = strings.ReplaceAll(defaultValue, "_utf8mb4", "")
+	defaultValue = strings.ReplaceAll(defaultValue, "\\\\", "\\")
+	defaultValue = strings.ReplaceAll(defaultValue, "\\'", "'")
+	if !generated && (ty == "char" || ty == "varchar" || ty == "text" || ty=="STRING") && !strings.HasPrefix(defaultValue, "'") && !strings.HasSuffix(defaultValue, "'") {
```

Comment:
Why are we checking for `STRING` if these are MySQL types, also I think the list isn't complete: https://www.w3schools.com/mysql/mysql_datatypes.asp

Reply:
It would be better to declare a list of string types and check if ty is present in the list and then do the sanitization